[2022-09-26T12:43:05.343+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2022-09-26T12:43:05.366+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2022-04-19T00:00:00+00:00 [queued]>
[2022-09-26T12:43:05.366+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T12:43:05.367+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 2
[2022-09-26T12:43:05.367+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T12:43:05.403+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): model.migrate_to_dbt.cast_obj> on 2022-04-19 00:00:00+00:00
[2022-09-26T12:43:05.416+0000] {standard_task_runner.py:54} INFO - Started process 2989 to run task
[2022-09-26T12:43:05.429+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_dag', 'model.migrate_to_dbt.cast_obj', 'scheduled__2022-04-19T00:00:00+00:00', '--job-id', '1972', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmpa0axwsph']
[2022-09-26T12:43:05.430+0000] {standard_task_runner.py:83} INFO - Job 1972: Subtask model.migrate_to_dbt.cast_obj
[2022-09-26T12:43:05.432+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dbt_dag.py
[2022-09-26T12:43:05.685+0000] {task_command.py:384} INFO - Running <TaskInstance: dbt_dag.model.migrate_to_dbt.cast_obj scheduled__2022-04-19T00:00:00+00:00 [running]> on host 3c654011b3f0
[2022-09-26T12:43:06.023+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=isaaclucky88@gmail.com
AIRFLOW_CTX_DAG_OWNER=astronomer
AIRFLOW_CTX_DAG_ID=dbt_dag
AIRFLOW_CTX_TASK_ID=model.migrate_to_dbt.cast_obj
AIRFLOW_CTX_EXECUTION_DATE=2022-04-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-19T00:00:00+00:00
[2022-09-26T12:43:06.026+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-09-26T12:43:06.027+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            cd /opt/***/migrate_to_dbt &&\n            dbt --no-write-json run --target prod --models cast_obj\n            ']
[2022-09-26T12:43:06.091+0000] {subprocess.py:86} INFO - Output:
[2022-09-26T12:43:06.104+0000] {subprocess.py:93} INFO - /bin/bash: line 3: dbt: command not found
[2022-09-26T12:43:06.107+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-09-26T12:43:06.173+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-09-26T12:43:06.184+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=dbt_dag, task_id=model.migrate_to_dbt.cast_obj, execution_date=20220419T000000, start_date=20220926T124305, end_date=20220926T124306
[2022-09-26T12:43:06.217+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 1972 for task model.migrate_to_dbt.cast_obj (Bash command failed. The command returned a non-zero exit code 127.; 2989)
[2022-09-26T12:43:06.261+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-09-26T12:43:06.769+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
